runtime/vm/FastJNI_sun_misc_Unsafe.cpp

```diff
@@ -48,4 +48,11 @@ J9_FAST_JNI_METHOD_TABLE(sun_misc_Unsafe)
 		J9_FAST_JNI_RETAIN_VM_ACCESS | J9_FAST_JNI_DO_NOT_WRAP_OBJECTS | J9_FAST_JNI_DO_NOT_PASS_RECEIVER)
 J9_FAST_JNI_METHOD_TABLE_END
 
+
+J9_FAST_JNI_METHOD_TABLE(jdk_internal_misc_Unsafe)
```

Comment:
Given this and the `J9_FAST_JNI_METHOD_TABLE(sun_misc_Unsafe)` table are identical now, we don't need to create the second copy.   The `sun_misc_Unsafe` table can be assigned to the `J9_FAST_JNI_CLASS("jdk/internal/misc/Unsafe", jdk_internal_misc_Unsafe)` class in FastJNI.cpp

Reply:
Super - that is a great idea - let me update the PR

Reply:
Done in e3c4fc6